python/sglang/srt/layers/torchao_utils.py

```diff
@@ -26,12 +26,11 @@ def apply_torchao_config_to_model(
         quantize_,
     )
     from torchao.quantization.observer import PerRow, PerTensor
-    from torchao.quantization.quant_api import _is_linear
 
     if filter_fn is None:
 
         def filter_fn(module, fqn):
-            return _is_linear(module) and "proj" in fqn
+            return "proj" in fqn
```

Comment:
@merrymercy this change? or the reordering of `apply_torchao_config_to_model` only works for torch native llama?

Reply:
I think it is this change.

Reply:
I picked the correct change in this PR https://github.com/sgl-project/sglang/pull/2495.
Please follow up with the correct fix for `_is_linear`.